src/accelerate/utils/operations.py

```diff
@@ -219,6 +222,12 @@ def gather(tensor):
     Returns:
         The same data structure as `tensor` with all tensors sent to the proper device.
     """
+    if not is_torch_distributed_available():
+        raise ModuleNotFoundError(
+            "torch was compiled without distributed support (torch.distributed.is_available() = False). "
+            "Please use a version which includes distributed support in order to call #gather()"
```

Comment:
```suggestion
            "`torch` was compiled without distributed support (`torch.distributed.is_available() == False`). "
            "Please install a version which includes distributed support in order to call `gather`."
```
Let's format these errs a little differently, otherwise LG2M!